app/src/debug/java/net/squanchy/support/system/DebugCurrentTime.kt

```diff
@@ -1,10 +1,37 @@
 package net.squanchy.support.system
 
+import android.content.Context
+import android.preference.PreferenceManager
+import androidx.core.content.edit
 import org.threeten.bp.ZonedDateTime
+import org.threeten.bp.format.DateTimeFormatter
 
-class DebugCurrentTime : CurrentTime {
+class DebugCurrentTime(context: Context) : CurrentTime {
```

Comment:
Maybe we should rename to `FreezableCurrentTime`?

Reply:
b7afb08